.travis.yml

```diff
@@ -25,14 +25,15 @@ matrix:
       script: |
         cargo install clippy
         cargo clippy --all -- -D clippy-pedantic
+        cd coresimd
+        cargo clippy --all -- -D clippy-pedantic
   allow_failures:
     - env: RUSTFMT=On TARGET=x86_64-unknown-linux-gnu NO_ADD=1
     - env: CLIPPY=On TARGET=x86_64-unknown-linux-gnu NO_ADD=1
 install:
   - if [ "$NO_ADD" == "" ]; then rustup target add $TARGET; fi
 
 script:
-  - cargo generate-lockfile
```

Comment:
`cargo generate-lockfile` only really works if all the crates are in a workspace, so I have added write permissions to the docker runs.